programs/editor/Tools.js

```diff
@@ -183,6 +186,36 @@ define("webodf/editor/Tools", [
                     saveButton.placeAt(toolbar);
                 }
 
+                // SaveAs
+                if (saveAsOdtFile) {
```

Comment:
This stuff really points towards the need to make this toolbar properly extensible somehow. I'd expect the first thing a certain type of 3rd party will want to do is to switch to their own chrome instead.

Oh well... it's following what the current Romans have already done with this file :smiling_imp: